app/src/main/java/org/oppia/app/options/AppLanguageActivity.kt

```diff
@@ -44,4 +50,9 @@ class AppLanguageActivity : InjectableAppCompatActivity() {
     setResult(REQUEST_CODE_APP_LANGUAGE, intent)
     finish()
   }
+
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    outState.putSerializable(KEY_LANGUAGE_SELECTION_ADAPTER ,appLanguageActivityPresenter.getLanguageSelectionAdapter())
```

Comment:
Spacing is incorrect there should be space after `,` and not before that:
`outState.putSerializable(KEY_LANGUAGE_SELECTION_ADAPTER, appLanguageActivityPresenter.getLanguageSelectionAdapter())`